xun/functions/store/disk.py

```diff
@@ -0,0 +1,121 @@
+from .store import Store
+from .store import StoreDriver
+from pathlib import Path
+import hashlib
+import pickle
+
+
+def key_hash(key):
+    pickled = pickle.dumps(key)
+    return hashlib.sha256(pickled).hexdigest()
+
+
+class Disk(Store):
+    class Driver(StoreDriver):
```

Comment:
Should maybe be named DiskDriver, to be consistent with the naming in diskcache.py and redis.py

Reply:
Also, in diskcache and redis, this inner class is defined after the outer class, not inside. Put outside for consistency?

Reply:
It's inside, because the driver is the important part. In for example the redis one, they are seperated, because both the store, and driver are complex